vic/drivers/cesm/src/cesm_put_data.c

```diff
@@ -0,0 +1,295 @@
+/******************************************************************************
+ * @section DESCRIPTION
+ *
+ * Collect coupled fields and export to coupler
+ *
+ * Sign convention: positive value <=> downward flux
+ * Units: see notes in code or seq_flds_mod.F90
+ *
+ * @section LICENSE
+ *
+ * The Variable Infiltration Capacity (VIC) macroscale hydrological model
+ * Copyright (C) 2014 The Land Surface Hydrology Group, Department of Civil
+ * and Environmental Engineering, University of Washington.
+ *
+ * The VIC model is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *****************************************************************************/
+
+#include <vic_driver_cesm.h>
+
+void
+vic_cesm_put_data()
+{
+    extern all_vars_struct    *all_vars;
+    extern atmos_data_struct  *atmos;
+    extern dmy_struct          dmy;
+    extern domain_struct       local_domain;
+    extern soil_con_struct    *soil_con;
+    extern veg_con_struct    **veg_con;
+    extern veg_lib_struct    **veg_lib;
+    extern l2x_data_struct    *l2x_vic;
+    extern x2l_data_struct    *x2l_vic;
+    extern global_param_struct global_param;
+    extern option_struct       options;
+
+    bool                       IsWet = false; // TODO: add lake fraction
```

Comment:
Any particular reason why lake fraction isn't included yet?  Is it fairly involved?  Or was this code change made before the image mode lake code was finished?